frontend/packages/integration-tests-cypress/tests/app/console-plugin-manifest.cy.ts

```diff
@@ -0,0 +1,273 @@
+import { safeLoadAll } from 'js-yaml';
+import { checkErrors } from '../../support';
+import { isLocalDevEnvironment } from '../../views/common';
+import { detailsPage } from '../../views/details-page';
+import { guidedTour } from '../../views/guided-tour';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+
+const PLUGIN_NAME = 'console-demo-plugin';
+const PLUGIN_PATH = '../../../dynamic-demo-plugin';
+const PLUGIN_PULL_SPEC = Cypress.env('PLUGIN_PULL_SPEC');
+
+const enableDemoPlugin = (enable: boolean) => {
+  // find console demo plugin and enable it
+  cy.visit('k8s/cluster/operator.openshift.io~v1~Console/cluster/console-plugins');
+  cy.url().should(
+    'include',
+    'k8s/cluster/operator.openshift.io~v1~Console/cluster/console-plugins',
+  );
+  cy.get('.co-resource-item__resource-name').byLegacyTestID(PLUGIN_NAME).should('be.visible');
+  cy.byLegacyTestID(PLUGIN_NAME)
+    .parents('tr')
+    .within(() => {
+      cy.byTestID('edit-console-plugin').contains(enable ? 'Disabled' : 'Enabled');
+      cy.byTestID('edit-console-plugin').click();
+    });
+  modal.shouldBeOpened();
+  cy.contains('Cancel');
+  modal.modalTitleShouldContain('Console plugin enablement');
+  cy.byTestID(enable ? 'Enable-radio-input' : 'Disable-radio-input').click();
+  modal.submit();
+  modal.shouldBeClosed();
+  cy.byLegacyTestID(PLUGIN_NAME)
+    .parents('tr')
+    .within(() => {
+      cy.byTestID('edit-console-plugin').contains(enable ? 'Enabled' : 'Disabled');
+    });
+  cy.log(`Running plugin test on ci using PLUGIN_PULL_SPEC: ${PLUGIN_PULL_SPEC}`);
+  cy.byTestID(`${PLUGIN_NAME}-status`)
+    .should('include.text', enable ? 'Loaded' : '-')
+    .then(() => {
+      if (!enable) {
+        cy.byLegacyTestID(PLUGIN_NAME).click();
+        detailsPage.titleShouldContain(PLUGIN_NAME);
+        detailsPage.clickPageActionFromDropdown('Delete ConsolePlugin');
+        modal.shouldBeOpened();
+        modal.submit();
+      }
+    });
+};
```

Comment:
instead of copying this helper, can we instead add this test to `demo-dynamic-plugin.cy.ts`?